explorer/src/Pos/Explorer/Web/Server.hs

```diff
@@ -640,6 +641,15 @@ epochSlotSearch epochIndex slotIndex = do
         errMsg :: Text
         errMsg = sformat ("No blocks on epoch "%build%" page "%build%" found!") epoch page
 
+-- | Get the @Page@s number from an @Epoch@ or throw an exception.
+getEpochPagesOrThrow
```

Comment:
If you want to move the function outside the function scope, move it into the `Helpers` section.